server/stage.go

```diff
@@ -59,7 +59,7 @@ func (s *Server) registerStageRoutes(g *echo.Group) {
 		var tasksPatched []*api.Task
 		for _, task := range tasks {
 			taskPatched, err := s.patchTaskStatus(ctx, task, &api.TaskStatusPatch{
-				ID:        task.ID,
+				IDList:    []int{task.ID},
```

Comment:
Shall we use the new way to update all tasks above?

Reply:
Let's do this in another PR, because this change effectively makes `func patchTaskStatus` patch a list of tasks rather than a single one.